js/hview.js

```diff
@@ -1081,11 +1073,24 @@ $(function () {
         }
 
         if (adbDevice && !adbDevice.disconnectedDevice) {
-            menu.push(null, {
-                text: "Disconnect",
-                icon: "ic_disconnect",
-                id: 3
-            })
+            menu.push(null,
+                {
+                    text: "Save hierarchy",
+                    icon: "ic_save",
+                    id: 1
+                },
+                null,
+                {
+                    text: "Refresh",
+                    icon: "ic_refresh",
+                    id: 2
+                },
+                null,
```

Comment:
These are related, probably a divider is not needed here

Reply:
Not a problem. I can remove this divider and send out a second revision. I will try to update this CL rather than create a new one if possible.

Reply:
Done.

Btw, the way to update a pull request through github is pretty cool. In case you don't know, all you need to do is git force push to you forked branch, and the pull request is automatically updated.